truffle/src/main/java/org/jruby/truffle/runtime/core/CoreLibrary.java

```diff
@@ -419,28 +389,9 @@ private void initializeConstants() {
         // BasicObject knows itself
         basicObjectClass.setConstant(null, "BasicObject", basicObjectClass);
 
-        // TODO(cs): this should be a separate exception
-        mathModule.setConstant(null, "DomainError", edomClass);
-
         objectClass.setConstant(null, "ARGV", argv);
 
-        final RubyString separator = RubyString.fromJavaString(stringClass, "/");
-        separator.freeze();
-
-        fileClass.setConstant(null, "SEPARATOR", separator);
-        fileClass.setConstant(null, "Separator", separator);
-
-        if (File.separatorChar == '\\') {
-            final RubyString altSeparator = RubyString.fromJavaString(stringClass, "\\");
-            altSeparator.freeze();
-
-            fileClass.setConstant(null, "ALT_SEPARATOR", altSeparator);
-        } else {
-            fileClass.setConstant(null, "ALT_SEPARATOR", nilObject);
-        }
-
-        fileClass.setConstant(null, "PATH_SEPARATOR", RubyString.fromJavaString(stringClass, File.pathSeparator));
-        fileClass.setConstant(null, "FNM_SYSCASE", 0);
+        rubiniusModule.setConstant(null, "UNDEFINED", rubiniusUndefined);
```

Comment:
Understood.  I was just raising it as an area we'll have to watch.  It may not be as simple as updating jnr-posix calls is all.